cql/src/main/java/io/stargate/cql/impl/CqlImpl.java

```diff
@@ -95,6 +101,35 @@ public void start() {
       }
     }
 
+    String additionalPorts = System.getProperty("stargate.cql.additional_ports");
+    if (additionalPorts != null) {
+      List<Server> additionalServers =
+          Arrays.stream(additionalPorts.split("\\s*,\\s*"))
+              .map(
+                  s -> {
+                    int port;
+                    try {
+                      port = Integer.parseInt(s);
+                    } catch (NumberFormatException e) {
+                      port = 0;
+                    }
+                    if (port <= 0) {
+                      logger.error(
+                          "Invalid port value found in property 'stargate.cql.additional_ports': {}",
+                          s);
+                    }
+                    return port;
+                  })
+              .filter(p -> p > 0)
+              .map(p -> builder.withSSL(false).withPort(p).build())
```

Comment:
Do we not want to support SSL with "additional" ports?

Reply:
Not for the purpose of this ticket. We could add a backlog issue.

Reply:
ok